From 4a418498898ea3e83b5b6e739ff2f63e77dcb87c Mon Sep 17 00:00:00 2001
From: alexda84 <69979639+alexda84@users.noreply.github.com>
Date: Fri, 15 Nov 2024 12:20:25 +0100
Subject: [PATCH] Update Pod.swift

---
 OmniBLE/OmnipodCommon/Pod.swift | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/OmniBLE/OmniBLE/OmnipodCommon/Pod.swift b/OmniBLE/OmniBLE/OmnipodCommon/Pod.swift
index d0daef2e..88334b23 100644
--- a/OmniBLE/OmniBLE/OmnipodCommon/Pod.swift
+++ b/OmniBLE/OmniBLE/OmnipodCommon/Pod.swift
@@ -79,7 +79,7 @@ public struct Pod {
     // Checked to verify it agrees with value returned by pod during the pairing process.
     public static let cannulaInsertionUnits = 0.5
 
-    public static let cannulaInsertionUnitsExtra = 0.0 // edit to add a fixed additional amount of insulin during cannula insertion
+    public static let cannulaInsertionUnitsExtra = 0.8 // edit to add a fixed additional amount of insulin during cannula insertion
 
     // Default and limits for expiration reminder alerts
     public static let defaultExpirationReminderOffset = TimeInterval(hours: 2)
