From db05ce752d5eecd55f9795bc83004bee5741adab Mon Sep 17 00:00:00 2001
From: alexda84 <69979639+alexda84@users.noreply.github.com>
Date: Mon, 14 Apr 2025 14:47:56 +0200
Subject: [PATCH] Update LoopCoreConstants.swift

---
 LoopCore/LoopCoreConstants.swift | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Loop/LoopCore/LoopCoreConstants.swift b/Loop/LoopCore/LoopCoreConstants.swift
index 814b3b6dc2..89439a9106 100644
--- a/Loop/LoopCore/LoopCoreConstants.swift
+++ b/Loop/LoopCore/LoopCoreConstants.swift
@@ -16,7 +16,7 @@ public enum LoopCoreConstants {
     /// The amount of time in the future a glucose value should be considered valid
     public static let futureGlucoseDataInterval = TimeInterval(minutes: 5)
 
-    public static let defaultCarbAbsorptionTimes: CarbStore.DefaultAbsorptionTimes = (fast: .minutes(30), medium: .hours(2), slow: .hours(5))
+    public static let defaultCarbAbsorptionTimes: CarbStore.DefaultAbsorptionTimes = (fast: .minutes(20), medium: .hours(2), slow: .hours(5))
 
     /// How much historical glucose to include in a dosing decision
     /// Somewhat arbitrary, but typical maximum visible in bolus glucose preview
